src/rad/resources/schemas/statistics-1.0.0.yaml

```
@@ -0,0 +1,69 @@
+%YAML 1.1
+---
+$schema: asdf://stsci.edu/datamodels/roman/schemas/rad_schema-1.0.0
+id: asdf://stsci.edu/datamodels/roman/schemas/statistics-1.0.0
+
+
+title: |
+  Basic Statistical Information
+
+type: object
+properties:
+  zodiacal_light:
+    title: Estimated Zodiacal Light Surface Brightness (MJy/sr)
+    description: |
+      A model estimate of the Zodiacal light
+      contribution to the sky background in units of MegaJanskys
+      per steradian (MJy/sr).
+    type: number
+    sdf:
```

```suggestion
    unit: "MJy.sr**-1"
    sdf:
```
@PaulHuwe .  We should include documentation of the unit that isn't enforced.  We should do that for the other new unit-carrying columns below as well (image_median, image_rms).  This follows Brett's suggestion here: https://github.com/spacetelescope/rad/pull/438/files

---

I have splintered this out into its own issue: #491 
It isn't really needed for the mid-release, as it won't affect created files.